app/views/layouts/_application_content.html.erb

```diff
@@ -2,13 +2,6 @@
   <div id="content">
     <%= notifications %>
     <div id="breadcrumb">
-      <% unless @page_header.blank? %>
-        <div class="row form-group">
-          <h1 class="col-md-8">
-            <%=h @page_header %>
-          </h1>
-        </div>
-      <% end %>
```

Comment:
Looks like pages that don't have a welcome page rely on this to show the title. One example I found was for ansible variables in the ansible plugin, but there are probably others. this is because we set `@welcome` to true even if we failed to render the welcome page:
https://github.com/theforeman/foreman/blob/develop/app/controllers/application_controller.rb#L33-L34

Reply:
I've added: https://github.com/theforeman/foreman/pull/8040/files#diff-4cff928ae5dfd3fb850d4e1483656430b60a8431c32ce22ece18e7a348cfe22eR88
I think it is more readable than what we had and it should solve the problem for those pages.
Although I'm not sure, we should have two ways of displaying title (one in react component and one in erb/helper) :thinking:

Reply:
perhaps we should just reset `@welcome = false` in case the `render 'welcome'` fails?

Reply:
> perhaps we should just reset `@welcome = false` in case the `render 'welcome'` fails?

We definitelly should have, but I'd argue not here. I was trying to keep the same behaviour. I've just missed, the template is used for welcome pages as well.

setting `@welcome = false` can affect the plugins and I'd guess there is no need to get it to 2.3 release.

Reply:
I'm opening a #8088 to do so.